.github/workflows/connectors_weekly_build.yml

```diff
@@ -1,4 +1,4 @@
-name: Connector Ops CI - Connectors Weekly Tests
```

Comment:
Ok. Ill be more attached to this since I added it in the first place 😅 

But there was two things I wanted to solve originally
1. Define which team owned a workflow
2. Mark which of our workflows were "Business Critical" (e.g. Connector Publish) vs "one off creations" (Label Github issues, terminate zombie build instance, etc)

So Im fine changing these but is there a way to still achieve at least #2?

Unfortuantely [we cant use folders](https://github.com/orgs/community/discussions/18055)

But perhaps we could either place the title slug at the end and try to shorten it
    1. `Connectors Weekly Tests [Ops CI]`
    1. `Connectors Weekly Tests - Critical Ops CI`

Reply:
@bnchrch  What do you think of using a `critical_` prefix on the workflow file name?
I don't find it useful to show that a workflow is critical in the check name on PR, we know it's critical when it has a *REQUIRED* state.
I'd find it useful to use the filename prefix so that we can easily identify without subfolder, the critical workflow we're maintaining while not impacting the CI checks readability.

Reply:
Great idea!